controllers/access_control.go

```diff
@@ -375,7 +383,7 @@ func (roleCreate AerospikeRoleCreateUpdate) createRole(client *as.Client, adminP
 		return fmt.Errorf("could not create role %s: %v", roleCreate.name, err)
 	}
 
-	err = client.CreateRole(adminPolicy, roleCreate.name, aerospikePrivileges, roleCreate.whitelist)
+	err = client.CreateRole(adminPolicy, roleCreate.name, aerospikePrivileges, roleCreate.whitelist, roleCreate.readQuota, roleCreate.writeQuota)
```

Comment:
It's better to pass `roleCreate` as a parameter here.

Reply:
Agree, but it's a part of different library.